docs/source/contributing/using_gitpod.md

```diff
@@ -0,0 +1,126 @@
+:::
+
+3. Grant GitHub / Gitpod integration permissions.
+
+    a) Go to: [https://gitpod.io/user/integrations](https://gitpod.io/user/integrations)
+
+    b) Select GitHub and then "edit permissions"
+
+    c) Select these permission: user:email, public_repo, repo, workflow
+
+:::{figure-md} Gitpod integration
+
+![gitpod_integration](gitpod/gitpod_integration.png)
+
+Gitpod [integration options](https://gitpod.io/user/integrations)
+:::
+
+4. Within Gitpod, create a ["New Workspace"](https://gitpod.io/workspaces).  Here you will want to select the forked pymc repo. If you don't see it, you can paste into the "Context URL" your forked repo path.  For example:  `https://github.com/reshamas/pymc`.  Then select "New Workspace".
+
+:::{figure-md} Gitpod workspace
+
+![gitpod_workspace](gitpod/gitpod_workspace.png)
+
+Gitpod [workspace](https://gitpod.io/workspaces)
+:::
+
+:::{note}
+Gitpod will pull a container and set up the workspace.  It will take a few minutes for the container to build.
+:::
+
+5. Once Gitpod is up and running, the interface is similar to a Visual Studio Code (VSC) interface, which will appear in your browser. You will observe installation notices in the terminal window.  This can take 5-10 minutes. Once that is complete, the terminal will indicate you are on the "(base)" environment on Gitpod with your forked repo.
+
+    Here is an example:
+
+    ```bash
+    (base) gitpod@reshamas-pymc-0ygu5rf74md:/workspace/pymc$
+    ```
+
+:::{note}
+This working environment has been set up with [micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) which is a small, pure-C++ executable with enough functionalities to bootstrap fully functional conda-environments.
+:::
+
+6. Check that your git remotes are correct with `git remote -v` at the terminal.
+
+    Example:
+
+    ```bash
+    (base) gitpod@reshamas-pymc-0ygu5rf74md:/workspace/pymc$ git remote -v
+    origin  https://github.com/reshamas/pymc.git (fetch)
+    origin  https://github.com/reshamas/pymc.git (push)
+    upstream        https://github.com/pymc-devs/pymc.git (fetch)
+    upstream        https://github.com/pymc-devs/pymc.git (push)
+    (base) gitpod@reshamas-pymc-0ygu5rf74md:/workspace/pymc$
+    ```
+
+7. Check which version of python and pymc are being used at the terminal.
+
+    * Check version of pymc: `pip list | grep pymc`
+
+    Example:
+
+    ```bash
+    (base) gitpod@reshamas-pymc-vpfb4pvr90z:/workspace/pymc$ pip list | grep pymc
+    pymc                          5.1.0       /workspace/pymc
+    pymc-sphinx-theme             0.1
+    ```
+
+    * Check version of python: `python3 --version`
+
+    Example:
+
+    ```bash
+    (base) gitpod@reshamas-pymc-vpfb4pvr90z:/workspace/pymc$ python3 --version
+    Python 3.11.0
+    ```
+
+### Reminders
+
+:::{attention}
+At the terminal, before beginning work, remember to:
+
+1. Create a feature branch: `git checkout -b feature-branch`
```

Comment:
It should already be enforced by a pre-commit hook that you cannot commit to `main`. However, I noticed a typo in the pre-commit config, so I opened https://github.com/pymc-devs/pymc/pull/6565.

But actually the error message displayed by VS Code when pre-commit fails can be confusing.

It may be worthwhile documenting that in a subsequent PR.

Reply:
Issue opened here:
https://github.com/pymc-devs/pymc/issues/6566